### PR TITLE
Update top-level readme

### PR DESCRIPTION
**Pull request by aaltay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,34 @@
-# Examples of the Cloud Dataflow SDK for Java 
+# Examples of the Cloud Dataflow SDK for Java and Python
```
Comment (davorbonaci) [inline README.md]: Google Cloud Dataflow Examples

File: README.md [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
 processing pipelines. This repository hosts example pipelines that use the [Cloud
-Dataflow SDK for Java](https://cloud.google.com/dataflow/model/programming-model) and demonstrate the basic functionality of Google Cloud Dataflow. 
+Dataflow SDK](https://cloud.google.com/dataflow/model/programming-model) and demonstrate the basic functionality of Google Cloud Dataflow. 
 
 ## Word Count
 
 A good starting point for new users is our set of
-[word count](https://github.com/GoogleCloudPlatform/DataflowJavaSDK-examples/blob/master/src/main/java/com/google/cloud/dataflow/examples) examples, which computes word frequencies.  This series of four successively more detailed pipelines is described in detail in the accompanying [walkthrough](https://cloud.google.com/dataflow/examples/wordcount-example).
+word count ([java](java/examples/src/main/java/com/google/cloud/dataflow/examples), [python](python/dataflow_examples)) examples, which computes word frequencies.  This series of four successively more detailed pipelines is described in detail in the accompanying [walkthrough](https://cloud.google.com/dataflow/examples/wordcount-example).
```
Comment (davorbonaci) [inline README.md]: computes -> compute

Comment (davorbonaci) [inline README.md]: walkthrough link should go to Beam, not Dataflow, because Dataflow one is 1.x only

File: README.md [hunk 3]
```diff
@@ -1,32 +1,34 @@
...
 
-1. [`MinimalWordCount`](https://github.com/GoogleCloudPlatform/DataflowJavaSDK-examples/blob/master/src/main/java/com/google/cloud/dataflow/examples/MinimalWordCount.java) is the simplest word count pipeline and introduces basic concepts like [Pipelines](https://cloud.google.com/dataflow/model/pipelines),
+1. `MinimalWordCount`([java](java/examples/src/main/java/com/google/cloud/dataflow/examples/MinimalWordCount.java), [python](python/dataflow_examples/wordcount_minimal.py)) is the simplest word count pipeline and introduces basic concepts like [Pipelines](https://cloud.google.com/dataflow/model/pipelines),
```
Comment (davorbonaci) [inline README.md]: Beam site links

File: README.md [hunk 4]
```diff
@@ -1,32 +1,34 @@
...
 
-1. [`WordCount`](https://github.com/GoogleCloudPlatform/DataflowJavaSDK-examples/blob/master/src/main/java/com/google/cloud/dataflow/examples/WordCount.java) introduces Dataflow best practices like [PipelineOptions](https://cloud.google.com/dataflow/pipelines/constructing-your-pipeline#Creating) and custom [PTransforms](https://cloud.google.com/dataflow/model/composite-transforms).
+1. `WordCount`([java](java/examples/src/main/java/com/google/cloud/dataflow/examples/WordCount.java), [python](python/dataflow_examples/wordcount.py)) introduces Dataflow best practices like [PipelineOptions](https://cloud.google.com/dataflow/pipelines/constructing-your-pipeline#Creating) and custom [PTransforms](https://cloud.google.com/dataflow/model/composite-transforms).
 
-1. [`DebuggingWordCount`](https://github.com/GoogleCloudPlatform/DataflowJavaSDK-examples/blob/master/src/main/java/com/google/cloud/dataflow/examples/DebuggingWordCount.java)
+1. `DebuggingWordCount`([java](java/examples/src/main/java/com/google/cloud/dataflow/examples/DebuggingWordCount.java), [python](python/dataflow_examples/wordcount_debugging.py))
```
Comment (davorbonaci) [inline README.md]: aggregators -> metrics?

File: README.md [hunk 5]
```diff
@@ -1,32 +1,34 @@
...
 
-1. [`WindowedWordCount`](https://github.com/GoogleCloudPlatform/DataflowJavaSDK-examples/blob/master/src/main/java/com/google/cloud/dataflow/examples/WindowedWordCount.java) shows how to run the same pipeline over either unbounded PCollections in streaming mode or bounded PCollections in batch mode.
+1. `WindowedWordCount`([java](java/examples/src/main/java/com/google/cloud/dataflow/examples/WindowedWordCount.java)) shows how to run the same pipeline over either unbounded PCollections in streaming mode or bounded PCollections in batch mode.
 
 ## Building and Running
 
-The examples in this repository can be built and executed from the root directory by running:
+### Java
+
+The examples in this repository can be built and executed from the `java` directory by running:
```
Comment (davorbonaci) [inline README.md]: repository is no longer accurate. example module?

java directory isn't accurate either. you have to be in a subfolder of java.

File: README.md [hunk 6]
```diff
@@ -96,6 +113,7 @@ For more information on Google Cloud Dataflow, see the following resources:
 * [Google Cloud Dataflow](https://cloud.google.com/dataflow/)
```
Comment (davorbonaci) [inline README.md]: Apache Beam link?

File: README.md [hunk 7]
```diff
@@ -96,6 +113,7 @@ For more information on Google Cloud Dataflow, see the following resources:
...
 * [Dataflow Concepts and Programming Model](https://cloud.google.com/dataflow/model/programming-model)
```
Comment (davorbonaci) [inline README.md]: should point to beam's programming model, likewise in all links.

**Comment by aaltay** [comment]
Thank you @davorbonaci, PTAL.

**Pull request merged**